src/khoj/routers/storage.py

```diff
@@ -33,3 +33,25 @@ def upload_image(image: bytes, user_id: uuid.UUID):
     except Exception as e:
         logger.error(f"Failed to upload image to S3: {e}")
         return None
+
+
+AWS_USER_UPLOADED_IMAGES = os.getenv("AWS_USER_UPLOADED_IMAGES")
```

Comment:
Let's make the usage a bit more obvious?

```suggestion
AWS_USER_UPLOADED_IMAGES_BUCKET_NAME = os.getenv("AWS_USER_UPLOADED_IMAGES_BUCKET_NAME")
```